weather/src/main/java/com/weather/core/entity/WeatherEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.weather.core.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.weather.core.repository.deserialozer.WeatherJsonDeserializer;
+import java.io.Serializable;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder
+@Getter
+@Setter
```

Comment:
Setter is not needed in case builder please google how to map Jackson builder to Lombok builder.